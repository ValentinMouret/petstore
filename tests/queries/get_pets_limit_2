http get http://localhost:4010/pets?limit=2
