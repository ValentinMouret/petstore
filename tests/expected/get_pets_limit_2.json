[
  {
    "id": 0,
    "name": "cat",
    "tag": "lazy"
  },
  {
    "id": 1,
    "name": "dog",
    "tag": "good"
  }
]
